cypress/e2e/basicTests.cy.ts

```diff
@@ -0,0 +1,106 @@
+/// <reference types="cypress" />
+
+//Admin empty export to make this spec file a module
+export { }
+
+const restaurantPlacardSelector = 'li.place-card__list-item'
+const dropdownSelector = '.location-dropdown__button-text'
+const dropdownOptionsSelector = 'li button.location-dropdown__list-button'
+const footerSelector = '.footer'
+const placeCardCitySelector = '.place-card__city'
+const linkOfRestaurantSelector = '.place-card__name-link'
```

Comment:
Are you familiar with the Cypress practices for [query priority](https://testing-library.com/docs/queries/about/#priority)?

We should avoid using class selectors like this and try to access the content in the order of 1. accessible queries, 2. semantic queries, and 3. test ids.

Reply:
Thanks. I will study about this and make the changes accordingly. I was previously unaware of this practice. Thank you for pointing it out. :)